### PR TITLE
Aria

### PR DESCRIPTION
**Pull request by ShahirAliGit**
Final aria labels made

### CODE REVIEW

**Review by jhbeeber — APPROVED** [review]
LGTM! Crucial for the accessibility of the website. Good job!

File: app/views/users/activityhistory.html.erb [hunk 1]
```diff
@@ -3,20 +3,20 @@
...
   <div style="margin-bottom: 10px;">
-    <%= label_tag(:end_date, "End date:") %>
-    <%= date_field_tag(:end_date, params[:end_date]) %>
+    <%= label_tag(:end_date, "End date:", aria: {label: "End Date Selector"}) %>
+    <%= date_field_tag(:end_date, params[:end_date], aria: {label: "End Date Selection"}) %>
   </div>
   <div style="margin-bottom: 10px;">
     <%= submit_tag("Filter") %>
-    <%= button_tag("Reset", type: 'button', onclick: "window.location.href='#{user_history_activity_path(@user)}';") %>
+    <%= button_tag("Reset", type: 'button', onclick: "window.location.href='#{user_history_activity_path(@user)}';", aria: {label: "Filter Button"}) %>
```
Comment (CerebralDatabank) [inline app/views/users/activityhistory.html.erb:19]: I don't think this is labelled correctly. (This same error is on other places as well, please make sure you get all of them)

**Review by CerebralDatabank — CHANGES_REQUESTED** [review]
LGTM except for the comment I put, so just fix that and it should be good

**Review by annapachecohartman — APPROVED** [review]
Looks good to me, only thing is where did the Help button on the nav bar go? 
Screenshot 2024-03-31 at 6 32 27 PM

**Review by CerebralDatabank — COMMENTED** [review]
Approved

**Pull request merged**